validator/validator-main.protoascii

```diff
@@ -3145,6 +3145,19 @@ tags: {
     }
   }
 }
```

Comment:
This is a guess based on the above config - let me know if this needs to be different.